packages/scandipwa/src/component/MyAccountAddressPopup/MyAccountAddressPopup.container.js

```diff
@@ -116,7 +116,7 @@ export class MyAccountAddressPopupContainer extends PureComponent {
 
     showAddressNotification(status, operation) {
         const { showSuccessNotification, showErrorNotification } = this.props;
-        const message = __('You saved the address', operation);
+        const message = __(`You ${operation} the address`);
```

Comment:
Please don't put the variables in the translatable strings. This makes them untranslatable.
If you have to put the variable there, please use the placeholder `%s`.

Also, I don't see how this would solve the issue, since you didn't change anything. Does the `operation` variable contains the correct operation?